### PR TITLE
improve process list item

### PR DESCRIPTION
**Pull request by der-john**
This is another part of #2681 - list items are improved, both content-wise and styling.

Among others, I incorporated parts of a design for `list-item` components all across a3 that Nico Di Marco made in [his meinberlin mockup](https://marvelapp.com/1gc2d4i/screen/14193218).

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/ListItem.html [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+        <div class="list-item-author" data-ng-if="participationStartDate && participationEndDate">{{ "TR__PARTICIPATION_DATES" | translate:{
                 startDate: participationStartDate,
                 endDate: participationEndDate}
-            }}</li>
-            <li class="meta-list-item" data-ng-if="locationText"><i class="meta-list-meta-item-icon icon-pin-detail"></i> {{locationText | translate }}</li>
+            }}</div>
+        <ul class="list-item-meta meta-list">
+            <li class="meta-list-item" data-ng-if="processName">{{processName | translate}}</li>
         </ul>
+        <div class="list-item-text">{{shortDesc | translate }}</div>
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/ListItem.html]: This element needs to be documented

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_list_item.scss [hunk 2]
```diff
@@ -48,8 +48,9 @@ $thumbnail-width: 105px;
     @include container;
     @include rem(padding, 1.25em $padding);
     display: block;
-    border-bottom: 2px solid $color-structure-normal;
     position: relative;
+    margin-bottom: 1em;
+    box-shadow: 0 0 1em $color-structure-introvert;
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_list_item.scss]: Why was this changed?

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/ListItem.html (hunk 1 above)
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/ListItem.html]: cf. #2714.

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_list_item.scss (hunk 2 above)
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_list_item.scss]: @xi and I discussed the matter. I originally thought we should follow Nico Di Marco's design but it turned out to be unclear.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/ListItem.html [hunk 3]
```diff
@@ -1,18 +1,18 @@
-<a class="list-item {{selectedState}}" data-ng-class="{'m-contains-image': picture}" href="{{path | adhResourceUrl}}">
+<a class="list-item is-not-selected" data-ng-class="{'m-contains-image': picture}" href="{{path | adhResourceUrl}}">
```
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Process/ListItem.html]: @xi and I discussed it. It should disappear.
